### ISSUE TITLE
mobilenet batch norm effect single image's prediction

### ISSUE THREAD

**Issue by WillLiGitHub** [tensorflow/models#3590]
I finetune mobilenet with retrained ckpt file, I feed batch images to model when training , the train result is good, but when i feed single image to trained model, the result is bad, single images' predict distribution similar to each other. I guess the moving mean and moving variable(batch norm parameters) change when i feed the model with single image!

**Comment by AloisChezAccenture** [comment]
Same problem with batch norm. 
When I try to freeze the batch norm parameters using 
```
variable_averages = tf.train.ExponentialMovingAverage(0.0)
    ema_variables_to_restore = variable_averages.variables_to_restore()
    with tf.Session() as sess:
      read_saver = tf.train.Saver(ema_variables_to_restore)
      read_saver.restore(sess, current_checkpoint_file)
```

I get 

> NotFoundError (see above for traceback): Key MobilenetV1/Conv2d_8_depthwise/BatchNorm/beta/ExponentialMovingAverage not found in checkpoint

It seems that exponential moving average is not saved in the checkpoint file.
I have tried to use tf.train.Saver(tf.global_variables()) instead of just tf.train.Saver() as suggested in the link below but it did not solve the problem :/

https://stackoverflow.com/questions/45800871/tensorflow-save-restore-batch-norm

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
